privacyidea/lib/importotp.py

```diff
@@ -630,7 +632,9 @@ def export_pskc(tokenobj_list, psk=None):
         otpkey = tokenobj.token.get_otpkey().getKey()
         try:
             if tokenobj.type.lower() in ["totp", "hotp"]:
-                encrypted_otpkey = aes_encrypt_b64(psk, binascii.unhexlify(otpkey))
+                encrypted_otpkey = to_unicode(aes_encrypt_b64(psk, binascii.unhexlify(otpkey)))
+            elif tokenobj.type.lower() in ["pw"]:
+                encrypted_otpkey = to_unicode(aes_encrypt_b64(psk, otpkey))
             else:
                 encrypted_otpkey = aes_encrypt_b64(psk, otpkey)
```

Comment:
Wouldn't it be sensible to also apply ``to_unicode`` here? Otherwise, I think we'll end up with string representations of base64 bytestrings in the XML, like ``<xenc:CipherValue>b'ABAB...'</xenc:CipherValue>``

Reply:
WIll check.

Reply:
No, the decryption also assumes, that this would be a bytestring.

Reply:
done in d367ea32